backend/conferences/types.py

```diff
@@ -58,12 +64,30 @@ class ConferenceType(DjangoObjectType):
     topics = graphene.NonNull(graphene.List(graphene.NonNull(TopicType)))
     languages = graphene.NonNull(graphene.List(graphene.NonNull(LanguageType)))
     durations = graphene.NonNull(graphene.List(graphene.NonNull(DurationType)))
+    schedule = graphene.NonNull(
+        graphene.List(graphene.NonNull(ModelScheduleItemType)),
+        date=graphene.Date()
+    )
 
     timezone = graphene.String()
 
     def resolve_timezone(self, info):
         return str(self.timezone)
 
+    def resolve_schedule(self, info, date=None, room=None):
```

Comment:
are you using room? I don't think we need that filtering for now anyway :)

Reply:
I think I forgot to remove it

Reply:
Done